### PR TITLE
Artur dev models2

### PR DESCRIPTION
**Pull request by arturbartkowicz**

### CODE REVIEW

File: test/fixtures/employees.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (Axel1991) [inline test/fixtures/employees.yml]: Remove File

File: test/fixtures/employers.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (Axel1991) [inline test/fixtures/employers.yml]: Remove File

File: test/models/employee_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Axel1991) [inline test/models/employee_test.rb]: Remove File

File: test/models/employer_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Axel1991) [inline test/models/employer_test.rb]: Remove File

File: db/migrate/20191008202540_devise_create_employers.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
```
[the remark this answers is not in the document]
Comment (Axel1991) [reply db/migrate/20191008202540_devise_create_employers.rb]: Please uncomment whole Confirmable section

File: db/migrate/20191008202433_devise_create_employees.rb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
```
[the remark this answers is not in the document]
Comment (Axel1991) [reply db/migrate/20191008202433_devise_create_employees.rb]: Please uncomment whole Confirmable section

File: db/migrate/20191008202540_devise_create_employers.rb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
```
[the remark this answers is not in the document]
Comment (TiagoSoares) [reply db/migrate/20191008202540_devise_create_employers.rb]: @arturbartkowicz

**Pull request merged**